config/config.go

```diff
@@ -87,6 +82,40 @@ type InitOpts struct {
 	MaxFileSize       uint64
 	ComputeProviderID int
 	Throttle          bool
+	Scrypt            ScryptParams
+}
+
+type ScryptParams struct {
+	N, R, P uint32
+}
+
+func (p *ScryptParams) Validate() error {
+	if p.N == 0 {
+		return errors.New("scrypt parameter N cannot be 0")
+	}
+	if p.R == 0 {
+		return errors.New("scrypt parameter r cannot be 0")
+	}
+	if p.P == 0 {
+		return errors.New("scrypt parameter p cannot be 0")
+	}
+	return nil
+}
+
+func DefaultPowScryptParams() ScryptParams {
```

Comment:
NIT `DefaultPowParams` and `DefaultLabelParams` is shorter and makes confusing the two less likely. Scrypt is already in the type.